app/services/map-staging-cms-reductions.js

```diff
@@ -0,0 +1,61 @@
+const getStagingCmsReductions = require('./data/get-staging-cms-reductions')
+const getWorkloadOwnerId = require('./data/get-app-workload-owner-id')
+const getReductionReasonFromCode = require('./data/get-reduction-reason-from-code')
+const reductionStatus = require('../constants/reduction-status')
+
+module.exports = function () {
+  return getStagingCmsReductions()
+  .then(function (cmsReductions) {
+    var stgReductions = []
+    var promises = []
+
+    if (cmsReductions) {
+      cmsReductions.forEach(function (cmsReduction) {
+        promises.push(
+          getReductionReasonFromCode(cmsReduction.contactCode)
+          .then(function (reductionReason) {
+            return getWorkloadOwnerId(cmsReduction.contactStaffKey, cmsReduction.contactTeamKey)
+            .then(function (contactWorkloadOwnerId) {
+              return getWorkloadOwnerId(cmsReduction.omKey, cmsReduction.omTeamKey)
+              .then(function (omWorkloadOwnerId) {
+                var startDate = new Date(cmsReduction.contactDate)
+                var endDate = new Date(startDate)
+                endDate.setDate(endDate.getDate() + 30)
+
+                var contactReduction = {
+                  contactId: cmsReduction.contactId,
+                  workloadOwnerId: contactWorkloadOwnerId,
+                  hours: reductionReason.fixedAllowanceHours,
+                  reductionReasonId: reductionReason.id,
+                  effectiveFrom: startDate,
+                  effectiveTo: endDate,
+                  status: reductionStatus.ACTIVE,
```

Comment:
Confirmed that the startDate will always the day of the extract.  Only situation where it could be more than 30 days previous is if the worker has not processed the extract for that amount of time.  In that case the second phase of the reduction worker, which checks the statuses of reductions, should pick this up and set the reduction to ARCHIVED.